packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -147,11 +147,42 @@ export const diffAndWriteFiles = async ({
   }
 };
 
+type TransformOptions = {
+  /**
+   * Determines whether a file should be copied
+   * @param {string} file path to a file
+   * @param {RegExp} fileForCopyRegExp default RegExp used for determination
+   */
+  isFileForCopy?: (file: string, fileForCopyRegExp: RegExp) => boolean;
+  /**
+   * Determines whether a file should be skiped (not copied/rendered).
+   * @param {string} file path to a file
+   */
+  isFileForSkip?: (file: string) => boolean;
+  /**
+   * Custom RegExp to determine which files should be copied without render
+   */
+  fileForCopyRegExp?: RegExp;
```

Comment:
Is it confusing to offer both `isFileForCopy` and `fileForCopyRegExp` options? If they needed to override the RegExp, they could technically do that with `isFileForCopy`.

Reply:
I thought about it if you want just to override default RegExp, without `isFileForCopy` usage. And it's possible that they will want to use our RegExp and add some additional logic, in this case I provided this regexp as the second parameter to the function

Reply:
It's problematically to extend our internal regexp if you need it, it was an original issue

Reply:
Agree, I think it's good as-is, especially with the additional doc comment you added to `isFileForCopy`.